packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -71,8 +71,9 @@ function selectNavigationMenus( select ) {
 	const args = [
 		'postType',
 		'wp_navigation',
-		{ per_page: -1, status: [ 'publish', 'draft' ] },
+		{ per_page: -1, status: 'publish,draft' },
```

Comment:
@adamziel Do you have any idea why this might be the case? Seems like a bug in the underlying implementation of Core Data.

Reply:
I looked into what might be going on here. It doesn't seem to make a difference in the final query before the rest api WP_Query happens. The comma separated string gets transformed into an array before the WP_Query. This image shows the status array query vs string query end up being identical.

<img width="1080" alt="Screenshot 2023-03-16 at 3 54 28 PM" src="https://user-images.githubusercontent.com/967608/225751331-04607ab8-197d-464b-bc95-8432e1947a46.png">

I dumped the $query_args with `die()` after it if it was a navigation request to see [the params here](https://github.com/WordPress/wordpress-develop/blob/6.1/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L361).